guides/v2.0/extension-dev-guide/test-module.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+group: extension-dev-guide
+subgroup: 4_Validate
+title: Test your component
+menu_title: Test your component
+menu_order: 2
+menu_node: 
+github_link: extension-dev-guide/test-module.md
+
+---
+
+##{{page.menu_title}}
+
+Test your component by deploying Magento Community Edition and adding the component to the project's <code>composer.json</code>. 
+
+{% highlight JSON %}
+"require": {
+    "magento/magento-composer-installer": "*",
+    "magento/product-community-edition": "2.0.0",
+    "foovendor/module-one": "0.1.1"
+},
+{% endhighlight %}
+
+Remember to [register](component-registration.html) your component as well, adding the location of your component. Confirm that your component works as expected and Magento functionality is not compromised.
```

Comment:
Please change the cross-ref to `[register]({{ site.gdurl }}extension-dev-guide/build/component-registration.html)`